doc/backend_vars.asciidoc

```diff
@@ -85,7 +85,7 @@ CDMODEL;see qemu -device ?;undef;Storage device for virtualized CD
 DELAYED_START;boolean;;delay vm cpu start until resume_vm() is called
 HDDFORMAT;;;HDD format
 HDDMODEL;see qemu -device ?;virtio-blk;Storage device for virtualized HDD.
-HDDMODEL_$i;see qemu -device ?;virtio-blk;Storage device for virtualized HDD. Overrides global HDDMODEL for HDD_$i
+HDDMODEL_$_;see qemu -device ?;virtio-blk;Storage device for virtualized HDD. Overrides global HDDMODEL for HDD_$i
```

Comment:
What is this change about?

Reply:
This is a necessary change as our CI checks include a crosscheck of variable references in documentation. As we do not reference `HDDMODEL_$i` anymore but now `HDDMODEL_$_` this is what we need to mention here in the doc table as well